markdown/dev/reference/api/part/attr/en.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Part.attr()
+---
+
+This `Part.attr()` method can be used to add attributes to the Part object.
+It calls `this.attributes.add()` under the hood, and returns the Part object.
+
+If the third parameter is set to `true` it will call `this.attributes.set()`
+instead, thereby overwriting the value of the attribute.
+
+## Signature
+
+```js
+Part Part.attr( 
+  string name,
+  mixed value,
+  bool overwrite = false
+)
+```
+
+<Tip compact>This method is chainable as it returns the `Part` object</Tip>
```

Comment:
If you want the parser to pickup on markup inside a custom tag, you should leave an empty line as explained here: https://freesewing.dev/guides/markdown/frequent-mistakes#using-custom-tag-components